03.1/middlwares/error.middlware.ts

```diff
@@ -22,6 +24,15 @@ const validationErrorMiddleware = (
 };
 
 const errorMiddleware = (err: any, req: Request, res: Response, next: NextFunction): void => {
+  const { method, query, body, originalUrl } = req;
+  const queryStr = JSON.stringify(query);
+  const bodyStr = JSON.stringify(body);
+
+  logger.error({
+    message: `${method} ${originalUrl} ${bodyStr} ${queryStr} ${err.message}`,
+    label: 'error-middleware',
+  });
+
   res.status(500).send(err);
```

Comment:
Not sure that it is a good idea to return error in case of internal server error (better to send 'internal server error' message but put error into log file)

Reply:
It makes sense, updated.